openmrs/src/main/resources/webapp/messages/messages.properties

```diff
@@ -151,3 +151,6 @@ openMRS.config.error.motechIDRequired=Motech ID is required.
 openMRS.config.error.patientIdentifierTypeExists=Patient Identifier type already in use.
 openMRS.config.info.motechID=The name of the OpenMRS identifier used by MOTECH. This must match the identifier that you will create in OpenMRS. The default is MOTECH Id.
 
+#Cohort query triggers
+openMRS.cohortQuery.uuid=CohortQuery UUID
+openMRS.cohortQuery.gotMember="Got member from cohortquery"
```

Comment:
remove quotation, why 'got' not 'get' ? Additionally this trigger name sounds bad. In ticket description there is an information about 'CohortQuery Report Member' task trigger